README.md

```diff
@@ -53,7 +53,7 @@ The world is simultaneously underwater and on fire, and people want to know what
 - [Google Cloud Computing](https://cloud.google.com/sustainability/) - Google are a long way from perfect, but Google Cloud advertises 100% renewable energy. They don't _just_ offset, they actually reduce the electricity usage too, and "strive to build the world’s most energy-efficient computing network by squeezing more out of every watt of power we consume." Their main rival, Amazon AWS, does not do that, and actively hosts oil and gas companies, who use the platform to find more places to dig. Switching to Google Cloud (or a host who in turn is on Google Cloud), is one of many ways to make your platform have less impact on the earth.
 - [Green-E Carbon Offsets](https://www.green-e.org/certified-resources/carbon-offsets) - Green-e Climate is an excellent resource for carbon offsets. The organization holds retailers accountable by monitoring how offsets are transacted and advertised in the retail market, protecting both the buyer and the seller.
 - [GreenMovi](https://www.greenmovi.com.br) - Earn discount coupons anytime, just download the app available for Android and iOS platforms, register, and use it every time you use a non-polluting means of transport and be rewarded with Carb Points.
-- [Leave Me Alone](https://leavemealone.app/save-the-planet/) - Emails contribute to 986,000 tonnes of CO2 being dumped into the atmosphere every day, that's 4g per email. Unsubscribe from unwanted subscription emails and reduce your carbon footprint.
+- [Leave Me Alone](https://leavemealone.app/save-the-planet/) - Emails contribute to 986,000 tonnes of CO2 being dumped into the atmosphere every day, that's 4g per email. Unsubscribe from unwanted subscription emails and let's make this habit of deleting spam/unwanted emails, as deleting most of these redundant emails can reduce the numbers of server running and hence reduce your carbon footprint.
```

Comment:
Instead, better to unsubscribe unwanted email spams - this can reduce time on deleting. Just a thought.